programs/serializers.py

```diff
@@ -349,6 +378,26 @@ def get_outcomes(self, program):
 
         return retval
 
+    def get_projections(self, program):
+        projection_serializer = EmploymentProjectionSerializer(instance=program.current_projections, many=True)
+        return projection_serializer.data
+
+    def get_careers(self, program):
+        return program.careers
+
+    def get_projection_totals(self, program):
+        obj = program.current_projections.aggregate(
```

Comment:
Consider adding projection year range somewhere in the serialized data so users know when this data is applicable